v3/lints/cabf_smime_br/lint_subscribers_crl_distribution_points_are_http_test.go

```diff
@@ -0,0 +1,40 @@
+package cabf_smime_br
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+func TestSubscriberCrlDistributionPointsAreHTTP(t *testing.T) {
+	testCases := []struct {
+		Name           string
+		InputFilename  string
+		ExpectedResult lint.LintStatus
+	}{
+		{
+			Name:           "pass - cert with HTTP CRL distribution point",
+			InputFilename:  "smime/subscriber_with_http_crl_distribution_point.pem",
+			ExpectedResult: lint.Pass,
+		},
+		{
+			Name:           "error - cert without a non-HTTP CRL distribution point",
+			InputFilename:  "smime/subscriber_with_non_http_crl_distribution_point.pem",
+			ExpectedResult: lint.Error,
+		},
```

Comment:
Could you add an expected details check to ensure the lint is failing for the right reasons in your test cases?

Reply:
Done.

Reply:
Done. Thanks for the review.